auditbeat/tests/system/test_file_integrity.py

```diff
@@ -79,6 +82,20 @@ def wrap_except(expr):
         return False
 
 
+def is_ubuntu_x86_64():
+    if platform.system() == 'Ubuntu' in platform.platform():
```

Comment:
Not sure I understand what this is intended to do.
Additionally, on an Ubuntu x86_64 22.04 distro the above functions don't contain `Ubuntu`:

```
>>> platform.platform()
'Linux-6.5.0-1016-gcp-x86_64-with-glibc2.35'
>>> platform.system()
'Linux'
>>> help(platform)
```